utils.py

```diff
@@ -369,24 +259,25 @@ def compute_hydration_energy(entry, parameters, platform_name="Reference"):
     atoms = [atom for atom in molecule.GetAtoms()]
     natoms = len(atoms)
 
+
+    # Create context for solvent system.
+    timestep = 2.0 * units.femtosecond
+    solvent_integrator = openmm.VerletIntegrator(timestep)
+
+
+    # Create context for vacuum system.
+    vacuum_integrator = openmm.VerletIntegrator(timestep)
+
     # Assign GBSA parameters.
     for (atom_index, atom) in enumerate(atoms):
         [charge, sigma, epsilon] = nonbonded_force.getParticleParameters(atom_index)
         atomtype = atom.GetStringData("gbsa_type") # GBSA atomtype
         radius = parameters['%s_%s' % (atomtype, 'radius')] * units.angstroms
-        print("The radius is %s" % str(radius))
         scalingFactor = parameters['%s_%s' % (atomtype, 'scalingFactor')]
-        print("The scaling factor is %s " % str(scalingFactor))
         gbsa_force.setParticleParameters(atom_index, [charge, radius, scalingFactor])
 
-    # Create context for solvent system.
-    timestep = 2.0 * units.femtosecond
-    solvent_integrator = openmm.VerletIntegrator(timestep)
-    solvent_context = openmm.Context(solvent_system, solvent_integrator, platform)
-
-    # Create context for vacuum system.
-    vacuum_integrator = openmm.VerletIntegrator(timestep)
-    vacuum_context = openmm.Context(vacuum_system, vacuum_integrator, platform)
+    solvent_context = openmm.Context(solvent_system, solvent_integrator)
+    vacuum_context = openmm.Context(vacuum_system, vacuum_integrator)
```

Comment:
This is a context creation without an explicit platform being set, which presumably might default to CUDA on some machines.  Could this be it?